tests/unit/CourtOrder.spec.ts

```diff
@@ -0,0 +1,173 @@
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import { createLocalVue, mount, shallowMount } from '@vue/test-utils'
+import { createPinia, setActivePinia } from 'pinia'
+import { useBusinessStore, useConfigurationStore, useRootStore } from '@/stores'
+import CourtOrder from '@/views/CourtOrder.vue'
+import { FileUploadPdf } from '@/components/common'
+import { CourtOrderPoa } from '@bcrs-shared-components/court-order-poa'
+import mockRouter from './mockRouter'
+import VueRouter from 'vue-router'
+import { CorpTypeCd } from '@bcrs-shared-components/corp-type-module'
+import SbcFeeSummary from 'sbc-common-components/src/components/SbcFeeSummary.vue'
+import vuetify from 'sbc-common-components/src/plugins/vuetify'
+
+// suppress various warnings:
+// - "Unknown custom element <affix>" warnings
+// - "$listeners is readonly"
+// - "Avoid mutating a prop directly"
+// ref: https://github.com/vuejs/vue-test-utils/issues/532
+Vue.config.silent = true
+
+Vue.use(Vuetify)
+
+const localVue = createLocalVue()
+localVue.use(VueRouter)
+setActivePinia(createPinia())
+const businessStore = useBusinessStore()
+const rootStore = useRootStore()
+const configurationStore = useConfigurationStore()
+
+describe('Court Order View', () => {
+  beforeAll(() => {
+    // mock the window.location.assign function
+    delete window.location
+    window.location = { assign: vi.fn() } as any
+
+    // set configurations
+    const configuration = {
+      'VUE_APP_AUTH_WEB_URL': 'https://auth.web.url/'
+    }
+    configurationStore.setConfiguration(configuration)
+
+    // set necessary session variables
+    sessionStorage.setItem('BASE_URL', 'https://base.url/')
+    sessionStorage.setItem('CURRENT_ACCOUNT', '{ "id": "2288" }')
+  })
+
+  beforeEach(() => {
+    // init store
+    rootStore.currentDate = '2020-03-04'
+    businessStore.setLegalType(CorpTypeCd.COOP)
+    businessStore.setLegalName('My Test Entity')
+    businessStore.setIdentifier('CP1234567')
+    businessStore.setFoundingDate('1971-05-12T00:00:00-00:00')
+    rootStore.filingData = []
+    rootStore.keycloakRoles = ['staff']
```

Comment:
Can you add a new describe block to validate that non-staff cannot access this filing?